kytos/core/auth.py

```diff
@@ -150,6 +228,22 @@ def register_core_auth_services(self):
         self.controller.api_server.register_core_endpoint(
             "auth/users/<uid>", self._update_user, methods=["PATCH"]
         )
+        self.controller.api_server.register_core_endpoint(
```

Comment:
Aldo, these are registering new methods for the same routes, you can replace the existing lines passing the new method that you need without having to add duplicated ones with a new method that will handle it, maybe this part you've only added to quickly test, when you need me to review again, let me know.

Reply:
Great progress